bulbs/contributions/models.py

```diff
@@ -159,14 +160,17 @@ class ContributorRole(Indexable):
     payment_type = models.IntegerField(choices=ROLE_PAYMENT_TYPES, default=MANUAL)
 
     def save(self, *args, **kwargs):
+        created = bool(self.pk is None)
         super(ContributorRole, self).save(*args, **kwargs)
-        if self.payment_type == 1:
-            self.create_feature_type_rates()
-
-    def create_feature_type_rates(self):
-        for feature_type in FeatureType.objects.all():
-            rate = FeatureTypeRate.objects.filter(role=self, feature_type=feature_type)
-            if not rate.exists():
+        self.create_feature_type_rates(created)
+        update_role_rates.delay(self)
```

Comment:
I've always avoided sending objects over Celery (to avoid serialization errors) and instead just sent IDs. Have you had success passing objects to Celery w/ Django projects?